src/signals/incident/components/form/FileInput/index.test.js

```diff
@@ -23,6 +20,8 @@ describe('Form component <FileInput />', () => {
   let parent;
 
   beforeEach(() => {
+    // jest.useFakeTimers();
```

Comment:
Don't you need to tell jest that it should use fake timers?

Reply:
gedaan